src/dstack/api/server/_runs.py

```diff
@@ -58,12 +58,26 @@ def get(self, project_name: str, run_name: str) -> Run:
 
     def get_plan(self, project_name: str, run_spec: RunSpec) -> RunPlan:
         body = GetRunPlanRequest(run_spec=run_spec)
-        resp = self._request(f"/api/project/{project_name}/runs/get_plan", body=body.json())
+        # client >= 0.18.18 / server <= 0.18.17 compatibility tweak
+        if not run_spec.configuration.privileged:
+            exclude = {"run_spec": {"configuration": {"privileged": True}}}
```

Comment:
Can be simply `exclude = {"run_spec": {"configuration": {"privileged"}}}`

Reply:
I was not sure, pyright nagged about incompatible type. Still nags with dict though :)